### ISSUE TITLE
Why is text justified?

### ISSUE THREAD

**Issue by LeBaux** [denysvitali/thebestmotherfuckingwebsite#33]
Justifying text on the web is bad practice. Why? Here is a nice summary:
https://designforhackers.com/blog/justify-text-html-css/

Center align is also not recommended by W3C for WCAG 2.0:

> Many people with cognitive disabilities have a great deal of trouble with blocks of text that are justified (aligned to both the left and the right margins). The spaces between words create "rivers of white" running down the page, which can make the text difficult for some people to read. This failure describes situations where this confusing text layout occurs. The best way to avoid this problem is not to create a text layout that is fully justified. https://www.w3.org/TR/WCAG20-TECHS/G169.html

Look at any major publication:
https://www.theguardian.com/international
https://www.bbc.com/news
https://www.washingtonpost.com
...basically, any sensibly designed website dealing with textual content.

Thank you.

**Comment by denysvitali** [comment]
Closing the issue since this has been fixed already a long time ago :+1:
